xml_output/taxfile-2018.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We've updated the `.gitignore` so it doesn't ignore this file anymore, so let's delete it.